uptane-standard.md

```diff
@@ -419,7 +419,7 @@ A repository's Timestamp role SHALL produce and sign metadata indicating whether
 
 ## Metadata structures {#meta_structures}
 
-Uptane's security guarantees all rely on properly created metadata that follow a designated structure. The Uptane standard **does not** mandate any particular format or encoding for the metadata as a whole. ASN.1 (with any encoding scheme like BER, DER, XER, etc.), JSON, XML, or any other encoding format that is capable of providing the required structure MAY be used.
+Uptane's security guarantees all rely on properly created metadata that follows a designated structure. The Uptane standard **does not** mandate any particular format or encoding for the metadata as a whole. ASN.1 (with any encoding scheme like BER, DER, XER, etc.), JSON, XML, or any other encoding format that is capable of providing the required structure MAY be used.
```

Comment:
"data" is plural so "follow" is correct. I go back and forth on how I really feel about that (I also often treat it as singular), but I think "follow" sounds better here.

Reply:
I go back and forth on this. You are right that, by grammatical rules, "data" is plural. However, its become common usage, I believe, to treat it in the singular form as well. In this case, I agree that "follow" sounds better.